lookout/style/format/descriptions.py

```diff
@@ -66,7 +66,7 @@ def describe_rule(rule: Rule, feature_extractor: FeatureExtractor) -> str:
         for group, nodes in grouped.items()
         for node, feature_names in nodes.items()
         for feature_name, splits in feature_names.items()]
-    return "%s\n\t→ y = %s (%.2f confidence, %d support)" % (
+    return "\t%s\n\t→ y = %s (%.2f confidence, %d support)" % (
```

Comment:
I think it makes more sense to keep the first line not indented in general (and to add the tab yourself if you want it indented in your special case). Ie in rule_stats, the web viewer etc this will deteriorate the result.

Reply:
ok, make sense.